modules/osdk-ansible-metrics.adoc

```diff
@@ -0,0 +1,225 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-monitoring-prometheus.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-ansible-metrics_{context}"]
+= Exposing custom metrics for Ansible-based Operators
+
+As an Operator author creating Ansible-based Operators, you can use the Operator SDK's `osdk_metrics` module to expose custom Operator and Operand metrics, emit events, and support logging.
+
+.Prerequisites
+
+* Ansible-based Operator generated using the Operator SDK
+* Prometheus Operator (deployed by default on {product-title} clusters)
```

Comment:
```suggestion
* Prometheus Operator deployed by default on {product-title} clusters
```

Or something similar. Per the IBM style guide, we should avoid using parentheses in general text. 
> Avoid using parentheses in general text because text in parentheses is decreased in importance.

Reply:
Changed to ", which is"